src/stack.py

```diff
@@ -0,0 +1,16 @@
+from linked_list import LinkedList
+
+
+class Stack(object):
+    """Make a new class Stack from using methods from Linked List"""
+    def __init__(self, data):
+        self.newLinkedList = LinkedList(data)
```

Comment:
This is an attribute that should not be directly accessed by users, so should begin with an underscore and look like:
```python
self._new_linked_list = LinkedList(data)
```